BinanceExchange.API/Endpoints.cs

```diff
@@ -201,14 +202,14 @@ public static BinanceEndpointData DepositAddress(DepositAddressRequest request)
             }
         }
 
-        private static string GenerateQueryStringFromData(IRequest request)
+        public static string GenerateQueryStringFromData(IRequest request)
```

Comment:
Why would you want this to be `public`?

Reply:
I used this static method at one of the TestMethods I'd written down to demonstrate the problem to you and because of that change the access level as public. 
But I've reverted it, I suppose the last version that I've recently committed seems OK for you.